app/models/article.rb

```diff
@@ -1,5 +1,6 @@
 class Article < ActiveRecord::Base
   has_many :comments, dependent: :destroy
```

Comment:
should have mentioned this earlier but your associations should have the inverse_of set as well. Makes it a little easier on active record.  see [here](http://guides.rubyonrails.org/association_basics.html#bi-directional-associations)
this one would be 

```ruby
has_many :comments, dependent: :destroy, inverse_of: :article
```